lib/postprocessing/image.js

```diff
@@ -12,13 +12,14 @@
 
 'use strict';
 
-const { OpenwhiskActionName, GenericError, RenditionFormatUnsupportedError } = require('@adobe/asset-compute-commons');
+const { OpenwhiskActionName, GenericError, RenditionFormatUnsupportedError, SourceCorruptError } = require('@adobe/asset-compute-commons');
 
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
 const { getAsset } = require('../storage');
 const { Dimensions } = require("./math");
+const unitConvert = require('css-unit-converter');
```

Comment:
Sizing information, if present in an SVG, may have units. And this unit may be something else than `pixels`, so we would need to convert. This module is for doing conversions.